src/components/core/AIChat/AIIcon.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { cx } from "cva";
+import { motion, MotionProps } from "motion/react";
+import { Icons } from "@/components/icons";
+
+export const AIIcon = ({ className = "" }: { className?: string }) => {
+  const transition: MotionProps["transition"] = {
```

Comment:
Do you want to just move this outside of the component. That way every render, it will know that the object reference has stayed the same so it shouldn't need to rebuild the animation.

Or you could just memo this whole component which I think would be equivalent.